src/ws/service/friendship_event_updates.rs

```diff
@@ -108,17 +107,6 @@ pub async fn handle_friendship_update(
     )
     .await?;
 
-    // Store the friendship event in the given room.
```

Comment:
This is no longer used since M1, right?

Reply:
Yes, that code was for retrocompatibility for friendships handled by Matrix. Can you confirm @Julieta11 ?

Reply:
In M1 we're still storing the event in the synapse room. This code was implemented for retrocompatibility to maintain the option to rollback to Matrix without losing any friendship interaction updates from M2 on.

Reply:
We won't be rolling back from M1 to Matrix for sure, if this code was meant for that, let's go ahead and remove it.